components/public-api-server/pkg/apiv1/tokens_test.go

```diff
@@ -464,16 +464,31 @@ func TestTokensService_DeletePersonalAccessToken(t *testing.T) {
 		require.Equal(t, connect.CodeInvalidArgument, connect.CodeOf(err))
 	})
 
-	t.Run("unimplemented when feature flag enabled", func(t *testing.T) {
-		serverMock, _, client := setupTokensService(t, withTokenFeatureEnabled)
+	t.Run("delete token", func(t *testing.T) {
+		serverMock, dbConn, client := setupTokensService(t, withTokenFeatureEnabled)
 
-		serverMock.EXPECT().GetLoggedInUser(gomock.Any()).Return(user, nil)
+		tokens := dbtest.CreatePersonalAccessTokenRecords(t, dbConn,
+			dbtest.NewPersonalAccessToken(t, db.PersonalAccessToken{
+				UserID: uuid.MustParse(user.ID),
+			}),
+		)
 
-		_, err := client.DeletePersonalAccessToken(context.Background(), connect.NewRequest(&v1.DeletePersonalAccessTokenRequest{
-			Id: uuid.New().String(),
+		serverMock.EXPECT().GetLoggedInUser(gomock.Any()).Return(user, nil).MaxTimes(3)
```

Comment:
```suggestion
		serverMock.EXPECT().GetLoggedInUser(gomock.Any()).Return(user, nil).Times(3)
```
We know exactly how many times it should be called from our test setup, so better to be explicit. This guards against a code change in the future where we perhaps remove the call to GetLoggedInUser, but the test would continue to pass